gilbreth_perception/config/parameters.yaml

```diff
@@ -11,6 +11,8 @@ parameters:
   cg_size: 0.03
   cg_thresh: 8.0
   descr_dis_thrd: 0.1
+  #fine_align
+  iteration: 10
```

Comment:
Does the alignment refinement iteration add significant time to the computation?

Reply:
On the server, it adds around 500ms on average.

Reply:
That's not too bad